components/schema-migrator/migrations/director/20220112165915_add_system_status_field_to_applications.up.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+ALTER TABLE applications
+    ADD COLUMN system_status varchar;
```

Comment:
In Postgres it's better to use `TEXT` which is more flexible when it comes to setting a length limit.